go.mod

```diff
@@ -1,34 +1,24 @@
 module github.com/CosmWasm/wasmd
 
-go 1.13
+go 1.14
 
 require (
 	github.com/CosmWasm/go-cosmwasm v0.8.1-0.20200604114456-1b2359bb7eb9
-	github.com/btcsuite/btcd v0.0.0-20190807005414-4063feeff79a // indirect
-	github.com/cosmos/cosmos-sdk v0.38.3
-	github.com/golang/mock v1.4.3 // indirect
+	github.com/cosmos/cosmos-sdk v0.34.4-0.20200530180557-ba70f4d4dc2e
```

Comment:
This is goz-phase-3 right?

Can you add that to the README? You merged in gaia, then upgraded the cosmos-sdk dep.
These numberings are not so clear so a comment in the README is helpful for people asking questions.

Reply:
I added the tag + version and also a note on `encoding`.